ci/run_easymesh_cert.sh

```diff
@@ -75,19 +81,12 @@ main() {
 
     info "Start running tests"
     run "$EASYMESH_CERT_PATH"/run_test_file.sh -o "$LOG_FOLDER" -d "$TARGET_DEVICE" "$TESTS" "$VERBOSE_OPT"
+    mv prplmesh.{ipk,buildinfo} "$LOG_FOLDER"
 
-    passed=$(find "$LOG_FOLDER"/PASS/ -maxdepth 1 -type d -printf x | wc -c)
-    failed=$(find "$LOG_FOLDER"/FAIL/ -maxdepth 1 -type d -printf x | wc -c)
-    total=$((failed+passed))
-    if [ "$VERBOSE" = "true" ]; then
-        info "Passed tests:"
-        ls "$LOG_FOLDER/PASS"
-        info "Failed tests:"
-        ls "$LOG_FOLDER/FAIL"
```

Comment:
This bit is *not* done in run_test_file.sh, and I think it's useful to do it.

Reply:
Actually I've been wanting to at least print the list of failing tests at the end of the job for some times now, I think it would be useful to do it from run_test_file.sh.